com/codecool/java/dao/DbTransactionsDAO.java

```diff
@@ -0,0 +1,189 @@
+package codecool.java.dao;
+
+import codecool.java.model.BasicConnectionPool;
+import codecool.java.model.Student;
+
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+public class DbTransactionsDAO implements TransactionsDAO{
+    private BasicConnectionPool pool;
+
+    public DbTransactionsDAO() throws SQLException, ClassNotFoundException {
+        Class.forName("org.postgresql.Driver");
+        this.pool = BasicConnectionPool.create("jdbc:postgresql://ec2-176-34-237-141.eu-west-1.compute.amazonaws.com:5432/dbc5jifafq3j1h?sslmode=require",
+                "utiuhfgjckzuoq", "17954f632e3663cbadb55550dd636f4c3a645ade56c3342ee89f71fc732c9672");
+    }
+
+    @Override
+    public List<QuestTransaction> loadAllNotApproved() throws SQLException {
+        Connection c = pool.getConnection();
+        List<QuestTransaction> unapprovedQuestsList = new ArrayList<>();
+        PreparedStatement ps = c.prepareStatement("SELECT * FROM student_quests WHERE date_approved IS NULL");
+        ResultSet rs = ps.executeQuery();
+        while(rs.next()) {
+            Integer coinsReceived = rs.getInt("coins_received");
+            Date dateAdded = rs.getDate("date_added");
+            Integer questId = rs.getInt("quest_id");
+            Integer userId = rs.getInt("user_id");
+            QuestTransaction questTransaction = new QuestTransaction(questId, userId, dateAdded, coinsReceived);
```

Comment:
```suggestion
            Transactions questTransaction = new QuestTransaction(questId, userId, dateAdded, coinsReceived);
```
consider use super class type here and at retrun type -> List<Transactions>

Reply:
good idea, I have changed so where possible